docs/kintsugi/governance.md

```diff
@@ -62,12 +62,14 @@ To vote on governance proposals, users must lock KINT with the Kintsugi parachai
 
 - The longer KINT are locked, the more vKINT are minted.
 - Lock periods:
-  - **Minimum: 1 week**
```

Comment:
can you please keep the minimum? Just to make it clear that this is the minim period

Reply:
That's the thing, it is not technically the minimum. Every week there is a checkpoint. If you create_lock, say, 2 days before such a checkpoint, you can reserve for 2 days.

Reply:
@sander2 create locks rounds it down though right?

Reply:
It rounds down to the checkpoint, yet. If you're 2 days from a checkpoint and you try to reserve for 3 days, you will actually lock it for 2

Reply:
I don't think that's correct, you wouldn't be able to lock it for 2 days. Assuming the minimum is a week and you try to lock for less than 7 days the following calculation would return 0: https://github.com/interlay/interbtc/blob/b4c86e64637abe7f711c527ae16d6749be65ea9a/crates/escrow/src/lib.rs#L356

Reply:
No it wouldn't - the argument to this function is not the length, but the end height